shippable.yml

```diff
@@ -43,5 +43,5 @@ integrations:
   # adding docker hub integration so that credentials are available to CI Job
   # http://docs.shippable.com/ci/push-gcr/
   hub:
-    - integrationName: gcr-integration
-      type: gcr
+    - integrationName: gcr-integration # we need to replace here with your Google Cloud integration name
```

Comment:
we need to change the integration name here to the new google cloud integration. Not sure what the name should be so havent' changed till yet